CONTRIBUTING.md

```diff
@@ -4,64 +4,67 @@ Thanks for contributing :)
 
 ## Building the app
 
-The application doesn't use Gradle and it might be hard to use some features of
-Android Studio.
+The application uses Gradle and can be used with Android Studio, but using Android Studio is not required and the project can be build and operated by just using Gradle.
 
-Fortunately, there's not many dependencies:
-- OpenJDK 8
+Fortunately, there are not many dependencies:
+- OpenJDK 17
+- Python 3
 - Android SDK: build tools (minimum `28.0.1`), platform `30`
-- Make sure to have the `$ANDROID_HOME` environment variable set.
+
+If you don't use Android Studio, you probably have to inform Gradle about the location of your Android SDK.
+To do this, create the file `local.properties` in the directory of this repository and paste `sdk.dir=<location_of_android_home>` e.g. `sdk.dir=/home/user/Android/Sdk`. Android Studio does this automatically.
```

Comment:
Is here an environment variable to accomplish this ? This would make the `shell.nix` easier to write.

Otherwise, the file could be created with a commented-out `sdk.dir` line to ease setup ?

Reply:
I realized the Android Gradle Plugin respects the ANDROID_HOME environment variable, so the file is just required when this variable is not set. I'll update the documentation, but since I don't use NixOS and am not able to properly test it, it would be great if you could do that.